api_tests/lib/create_actors.ts

```diff
@@ -13,15 +12,12 @@ export async function createActors(
 
     const listPromises: Promise<Actor>[] = [];
     for (const name of actorNames) {
-        const cndLogFile = path.join(
-            global.logRoot,
+        const cndLogFile = global.getLogFile([
```

Comment:
```suggestion
        const cndLogFile = global.getLogDir([
```
This returns the path to the `logs` dir, not file.

Reply:
Negative, this just returns a path and is supposed to be used for files and not directories.